gramjs/client/uploads.ts

```diff
@@ -96,11 +97,13 @@ const DISCONNECT_SLEEP = 1000;
 
 async function getFileBuffer(
     file: File | CustomFile,
-    fileSize: number
+    fileSize: number,
+    maxBufferSize?: number
 ): Promise<CustomBuffer> {
     const isBiggerThan2Gb = fileSize > 2 ** 31 - 1;
+    const usePath = maxBufferSize ? fileSize > maxBufferSize : false;
     const options: CustomBufferOptions = {};
-    if (isBiggerThan2Gb && file instanceof CustomFile) {
+    if ((usePath || isBiggerThan2Gb) && file instanceof CustomFile) {
```

Comment:
wouldn't make the max buffer 2GB be better and that way we can get rid of isBiggerThan2Gb